src/elements/common/error-boundary/DefaultError.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import ErrorMask from '../../../components/error-mask/ErrorMask';
+import messages from '../messages';
+import './DefaultError.scss';
+
+export interface DefaultErrorProps {
+    error?: Error;
+}
```

Comment:
You should call this `ErrorComponentProps` since `DefaultError` doesn't take props